obj/gebaeude.cc

```diff
@@ -371,7 +371,7 @@ void gebaeude_t::calc_image()
 
 image_id gebaeude_t::get_image() const
 {
-	if(env_t::hide_buildings!=0  &&  tile->has_image()) {
+	if(env_t::hide_buildings!=0 &&  tile->has_image()) {
```

Comment:
Please follow the simutrans coding rule, although it is not strictly followed...
https://github.com/teamhimeh/simutrans/blob/6cb27bec30e16457af8b8db6797fa5690ab3e038/documentation/coding_styles.txt#L232-L237

```suggestion
	if(env_t::hide_buildings!=0  &&  tile->has_image()) {
```